ceph-docker-build/config/definitions/ceph-docker-build.yml

```diff
@@ -0,0 +1,49 @@
+- job:
+    name: 'ceph-docker-build'
+    node: 'centos7 && x86_64 && small && !sepia'
+    concurrent: true
+    defaults: global
+    display-name: 'ceph-docker: image builds'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph-docker
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+
+    triggers:
+      - github
+      #- parameterized-timer: '@daily % CEPH_RELEASE=kraken'
```

Comment:
Whoops, this was an earlier thing, left over from debugging. I've dropped it and amended this.